tests/cli.rs

```diff
@@ -0,0 +1,169 @@
+use std::io::{Read, Write};
+use std::iter::repeat;
+use std::process::{Command, Stdio};
+
+fn run_script<T: AsRef<str>>(commands: &[T]) -> Vec<String> {
```

Comment:
Learned from: https://stackoverflow.com/questions/41179659/convert-vecstring-into-a-slice-of-str-in-rust